meta-lmp-bsp/dynamic-layers/freescale-layer/recipes-bsp/imx-mkimage/imx-boot_%.bbappend

```diff
@@ -15,7 +15,155 @@ do_compile[depends] = " \
     ${@bb.utils.contains('MACHINE_FEATURES', 'optee', 'virtual/optee-os:do_deploy', '', d)} \
 "
 
-do_deploy_append() {
-    # Also create imx-boot link with the machine name
-    ln -sf ${BOOT_CONFIG_MACHINE}-${IMAGE_IMXBOOT_TARGET} ${DEPLOYDIR}/${BOOT_NAME}-${MACHINE}
```

Comment:
It seems this was removed.

Reply:
didn't get your comment

Reply:
This line was added so we could have imx-boot-MACHINE, as you can see at https://ci.foundries.io/projects/lmp/builds/1170/build-imx8mmevk/. This is because the mfgtools scripts are using imx-boot-MACHINE as well, to avoid conflicts when downloading the built artifacts from multiple imx-based boards.

Reply:
as we have now two binaries, addressed this a bit different (check `do_deploy_append()`):
```
# Creating links for mfgtools scripts
ln -sf SPL-${BOOT_CONFIG_MACHINE}-${IMAGE_IMXBOOT_TARGET} SPL-${MACHINE}
ln -sf u-boot.itb-${BOOT_CONFIG_MACHINE}-${IMAGE_IMXBOOT_TARGET} u-boot-${MACHINE}.itb

```